scripts/faq.js

```diff
@@ -33,7 +33,9 @@ const getFaq = (lang) => {
 
 fs.readdir('./src/assets/i18n', (err, files) => {
   files.forEach(filename => {
-    getFaq(filename.split('.')[0]);
+    if(filename !== 'ru.json') {
+      getFaq(filename.split('.')[0]);
```

Comment:
I see no reason for this "fix". Remove [ru.json] instead. If we get a Russian translation of Zonemaster this file could be recreated.

[ru.json]: https://github.com/zonemaster/zonemaster-gui/blob/master/src/assets/i18n/ru.json